src/utils/dappMailer.js

```diff
@@ -410,4 +410,32 @@ module.exports = {
     // sendEmail(app, data);
     sendEmail(app, data);
   },
+
+  donationsCollected: (app, data) => {
+    Object.assign(data, {
+      template: 'notification',
+      subject: 'Giveth - Donations collected',
+      type: 'milestone-donations-collected',
+      secretIntro: `Your Milestone ${data.milestoneTitle} has been paid.`,
+      title: 'Milestone Donations Collected',
+      image: 'Giveth-milestone-review-approved-banner-email.png',
+      text: `
+        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user}</span></p>
+        <p>The following payments have been initiated for your Milestone <em>${
+          data.milestoneTitle
+        }</em>:</p>
+        <p></p>
+        ${data.conversation.payments.map(p => `<p>${p.amount / 10 ** 18} ${p.symbol}</p>`)}
+        <p></p>
+        <p>You can expect to see these payment(s) to arrive in your wallet <em>${
+          data.address
+        }</em> within 48 - 72 hrs.</p>
+      `,
+      cta: `See your Milestones`,
+      ctaRelativeUrl: `/campaigns/${data.campaignId}/milestones/${data.milestoneId}`,
+      unsubscribeType: 'donations-collected',
+      unsubscribeReason: `You receive this email because you are the recipient of a Milestone`,
+    });
+    sendEmail(app, data);
```

Comment:
@aminlatifi 
I wrote subjects and texts by myself, if you think there are some changes needed for texts let me know